pkg/healthcheck/healthcheck.go

```diff
@@ -1857,10 +1857,13 @@ func (hc *HealthChecker) checkClusterNetworks(ctx context.Context) error {
 		}
 	}
 	var badPodCIDRS []string
+	var nodesSkipped []string
+	var errStrings []string
```

Comment:
We could probably inline some of these declarations if we're not using the shorthand `:=`, it might make everything look a bit cleaner imo.

```suggestion
	var nodesSkipped, errStrings []string
```